Bluetooth: controller: llcp: inclusive naming in unittests

kruithofa opened the pull request:
There were some references to slave and master left in the unittests
for the refactored LLCP.
These are changed in respectively peripheral and central

Signed-off-by: Andries Kruithof <Andries.Kruithof@nordicsemi.no>

Commit:
Bluetooth: controller: llcp: inclusive naming in unittests

There were some references to slave and master left in the unittests
for the refactored LLCP.
These are changed in respectively peripheral and central

Signed-off-by: Andries Kruithof <Andries.Kruithof@nordicsemi.no>

Thalley commented on tests/bluetooth/controller/ctrl_conn_update/src/main.c:
```diff
@@ -2574,81 +2574,81 @@ void test_main(void)
 	ztest_test_suite(
-		sla_loc_no_param_req,
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_disallowed_no_param_req,
+		per_loc_no_param_req,
```
Personally I think the `per` abbreviation should be avoided to avoid clashes with periodic advertising which is also often abbreviated to `per`.

kruithofa replied:
per is now replaced with periph, cen with central

Thalley reviewed (changes requested):
A good step in the right direction, although I would strongly suggest avoiding the `per` abbreviation. 

There also seems to be a bit of a inconsistency between using `per`, `periph` and `peripheral` (ditto with `cen` and `central`. Any reason why not also fully write `peripheral` or use the existing abbreviate `periph`?

Commit:
bluetooth: controller: llcp: expand cen and per

The abbreviations cen and per may be confusing, so they are
expanded into central and peripheral

Signed-off-by: Andries Kruithof <Andries.Kruithof@nordicsemi.no>

Thalley commented on tests/bluetooth/controller/common/include/helper_pdu.h:
```diff
@@ -57,7 +57,7 @@ void helper_pdu_verify_ping_rsp(const char *file, uint32_t line, struct pdu_data
 void helper_pdu_verify_feature_req(const char *file, uint32_t line, struct pdu_data *pdu,
 				   void *param);
-void helper_pdu_verify_slave_feature_req(const char *file, uint32_t line, struct pdu_data *pdu,
+void helper_pdu_verify_peripheral_feature_req(const char *file, uint32_t line, struct pdu_data *pdu,
```
Keep in mind that by using `peripheral` there likely need to be some re-indendation and line breaks.

kruithofa replied:
There were, and I think I covered them all

Thalley replied:
This seems to be above the recommended (but not strict) 80 char line, and the `void *param` on next line is incorrectly indented :) Not sure if there's more like this

kruithofa replied:
80 is recommended, but the max is 100

kruithofa replied:
I see now that I missed the indentation; fixed.
Checked other places for correct indentation as well
